src/ui/components/results/verticalresultscomponent.js

```diff
@@ -152,10 +153,12 @@ export default class VerticalResultsComponent extends Component {
     }
   }
 
-  getVerticalURL (data) {
+  getVerticalURL (data = {}) {
     const verticalConfig = this._verticalsConfig.find(config => config.verticalKey === this.verticalKey) || {};
-    const verticalURL = verticalConfig.url || data.verticalURL || this.verticalKey + '.html';
-    return addParamsToUrl(verticalURL, { query: this.query });
+    const verticalURL = this._config.verticalURL || verticalConfig.url || data.verticalURL || this.verticalKey + '.html';
```

Comment:
nit but config.verticalURL doesn't exist yet

Reply:
ah ok, I'll remove it!